exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/marshal/MarshalerUtil.java

```diff
@@ -46,7 +46,7 @@
     JSON_AVAILABLE = jsonAvailable;
   }
 
-  static final byte[] EMPTY_BYTES = new byte[0];
+  public static final byte[] EMPTY_BYTES = new byte[0];
```

Comment:
Consider adding a method to access this.

Reply:
Just made it private instead